cpg_utils/flows/slack.py

```diff
@@ -0,0 +1,83 @@
+"""
+Sending notifications to Slack. To enable, create a channel, add "Seqr Loader" app
+into a channel with:
+
+/invite @Seqr Loader
```

Comment:
This is probably more generic now? Is there already a generic bot?

Reply:
Good question, I should look into creating a generic bot